test/source/mock/lib/api.ts

```diff
@@ -226,9 +226,10 @@ export class Api<REQ, RES> {
     if (body.length) {
       if (
         req.url!.startsWith('/upload/') || // gmail message send
-        req.url!.startsWith('/api/message/upload') || // flowcrypt.com/api pwd msg
+        req.url!.startsWith('/api/message/upload') || // flowcrypt.com/shared-tenant-fes pwd msg
```

Comment:
Is there still any `/api/message/upload` endpoint?

Reply:
I don't think so. let me remove it